dt-assets/js/shared-functions.js

```diff
@@ -575,6 +575,12 @@ window.SHAREDFUNCTIONS = {
     }
     return "";
   },
+  setCookie(cname, cvalue, exdays = 30 ) {
```

Comment:
@squigglybob save_json_cookie() also works on strings.

on a subdirectory multisite, would `/` set a cookie for the whole installation?

Reply:
yes it would and good point. Maybe I should add exdays to save_json_cookie

I think part of why I added setCookie was because there is a getCookie function but no setCookie function, but you're right it's not taking a path parameter

Reply:
I think we need a setCookie function which doesn't stringify strings by default.
It's the difference between saving a cookie that looks like this
![image](https://github.com/DiscipleTools/disciple-tools-theme/assets/9816214/bbe64adc-cd24-4a49-84b6-d182e0a83b94)

and this

![image](https://github.com/DiscipleTools/disciple-tools-theme/assets/9816214/9542944d-deab-4c02-b819-07621dff4100)

Reply:
what about adding exdays to save_json_cookie and refactoring that to use setCookie underneath